sdk/keyvault/keyvault-keys/tests/utils/recorder.ts

```diff
@@ -77,9 +75,7 @@ abstract class Recorder {
    * */
   protected filterSecrets(recording: string): string {
     let updatedRecording = recording;
-    let lastRecording = recording;
-    for (let k of Object.keys(replaceableVariables)) {
-      lastRecording = updatedRecording;
+    for (const k of Object.keys(replaceableVariables)) {
```

Comment:
Do you not use lastRecording later?

Reply:
Nope! It was a leftover from my copying and pasting from the original file. The original file is twice as big.